dev.js

```diff
@@ -17,13 +16,20 @@ const {
     externalizeClasses: {
       type: 'boolean',
     },
+    omitComponentClasses: {
+      type: 'boolean',
+    },
     usePixels: {
       type: 'boolean',
     },
   },
 });
-
-const uno = createGenerator({ presets: [presetWarp( { ...options, development: true } )] });
+const asyncPreset = await presetWarp({
+  ...options,
+  omitComponentClasses: true,
+  development: true,
```

Comment:
It looks like we intend to omit component classes here. The way I understand the plugin.js logic (see below), the external classes applied with `omitComponentClasses` flag will not be considered in the postprocessor.js. The constant below should be `true` for this to work, but it will be `false` when `options.extenalizeClasses` flag is `undefined` and `options.development` is `true`. For this to work we'd need to pass `externalizeClasses: true`.

```
const externalizeClasses = options.externalizeClasses ?? !options.development;
```

Reply:
Yes, I was testing this back and forward and left it like this by mistake. I pushed the fix for this now. I'll test locally when we have a meeting. It was confusing to leave it like that, sorry